homeassistant/components/rfxtrx/__init__.py

```diff
@@ -428,6 +434,14 @@ def get_device_id(device, data_bits=None):
     return (f"{device.packettype:x}", f"{device.subtype:x}", id_string)
 
 
+async def async_connect_auto_add(hass, entry_data, callback_fun):
```

Comment:
This doesn't need to be a coroutine function since we don't await inside.

Reply:
Done